opentok/opentok.py

```diff
@@ -444,6 +451,29 @@ def get_archives(self, offset=None, count=None):
         else:
             raise RequestError("An unexpected error occurred", response.status_code)
 
+    def force_disconnect(self, session_id, connection_id):
+        """Disconnects a participant from an OpenTok session. This is the server-side equivalent
+          to the forceDisconnect() method in OpenTok.js:
+          https://www.tokbox.com/developer/guides/moderation/js/#force_disconnect
+
+        :param String session_id: The session ID for the OpenTok session that the client you want
+          to disconnect is connected to.
+        :param String connection_id: The connection ID of the client you want to disconnect.
+        """
+        if not session_id or not connection_id:
+            raise OpenTokException(u('SessionId and ConnectionId are required'))
+
+        response = requests.delete(self.connection_url(session_id, connection_id), headers=self.api_headers(), proxies=self.proxies)
+
+        if response.status_code < 300:
+            pass
+        elif response.status_code == 403:
+            raise AuthError()
+        elif response.status_code == 404:
+            raise NotFoundError("Session or connection not found")
```

Comment:
I think this should be `u('Session or connection not found')` (and for consistency’s sake, single quotes). Same for the next raised exception.